### ISSUE TITLE
failed to pull docker image: Error response from daemon: manifest for public.ecr.aws/supabase/gotrue:vunspecified not found

### ISSUE THREAD

**Issue by tomchapin** [supabase/cli#2807]
Every time I try to do a supabase db pull, I am getting this error:

failed to pull docker image: Error response from daemon: manifest for public.ecr.aws/supabase/gotrue:vunspecified not found: manifest unknown: Requested image not found

Is anyone else running into this?

I have tried changing my IP address and using a VPN and all sorts of things. I have tried the latest homebrew version of the supabase CLI, and also the latest node version, and I run into this same issue on each of them.

When I try browsing to public.ecr.aws/supabase/gotrue directly, I get this error:

{"errors":[{"code":"DENIED","message":"Not Authorized"}]}

**Comment by tomchapin** [comment]
Ok, I did some digging. It looks like there is a way to over-ride the default registry that is used by specifying an environment variable.

I was able to finally get it all working, like so:
```
SUPABASE_INTERNAL_IMAGE_REGISTRY=docker.io supabase db pull
```

Someone on the supabase team might want to investigate why the public.ecr.aws registry is failing to serve the docker images?

**Comment by tomchapin** [comment]
Ok, I finally figured out what was going wrong.

The supabase folder had a ".temp" folder in it that had a number of text files specifying the version numbers of different images. The text file that was specifying the version number for gotrue just had a string that said "vunspecified".

This was apparently breaking everything.

I solved the problem by completely deleting the supabase/.temp/ folder in my project.
